crates/oxc_ast/src/serialize.rs

```diff
@@ -548,6 +548,16 @@ impl ESTree for JSXElementThisExpression<'_> {
     }
 }
 
+#[ast_meta]
+#[estree(ts_type = "Array<JSXAttributeItem>", raw_deser = "[]")]
+pub struct JSXOpeningFragmentAttributes<'b, T>(#[expect(dead_code)] pub &'b T);
```

Comment:
Can we remove the generic param? It seems unnecessary.

```suggestion
pub struct JSXOpeningFragmentAttributes<'b>(#[expect(dead_code)] pub &'b JSXOpeningFragment);
```